develop/parachains/maintenance/configure-asynchronous-backing.md

```diff
@@ -0,0 +1,310 @@
+
+!!!note
+    If starting a new parachain project, please use an async backing compatible template such as the [parachain template](https://github.com/paritytech/polkadot-sdk-parachain-template){target=\_blank}. The rollout process for async backing has three phases. Phases 1 and 2 below put new infrastructure in place. Then async backing is simply enabled in phase 3.
+
+## Prerequisite
+
+The relay chain needs to have async backing enabled so double-check that the relay chain configuration contains the following three parameters (especially when testing locally e.g. with [Zombienet](/develop/toolkit/parachains/spawn-chains/zombienet/get-started/){target=\_blank}):
+
+```rust title="runtimes/relay/polkadot/src/genesis_config_presets.rs"
+...
+"async_backing_params": {
+    "max_candidate_depth": 3,
+    "allowed_ancestry_len": 2
+},
+...
+```
+
+This can be found in the relay chain's runtime. You can see the Polkadot [`async_backing_params`](https://github.com/polkadot-fellows/runtimes/blob/d49a9f33d0ea85ce51c26c84a70b61624ec06901/relay/polkadot/src/genesis_config_presets.rs#L131-L134){target=\_blank} as an example. You also want to make sure that the `lookahead` in [`schedulerParams`](https://paritytech.github.io/polkadot-sdk/master/cumulus_primitives_core/relay_chain/struct.SchedulerParams.html){target=\_blank} is set to `3`. This can be found by querying the [`scheduler_params`](https://paritytech.github.io/polkadot-sdk/master/polkadot_runtime_parachains/configuration/struct.HostConfiguration.html#structfield.scheduler_params){target=\_blank} using the [`configuration.activeConfig()`](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc-polkadot.helixstreet.io#/chainstate) query in Polkadot JS.
+
+!!! warning 
+    The `lookahead` field must be set to 3; otherwise, parachain block times will degrade to worse than with sync backing!
+
+## Phase 1 - Update Parachain Runtime
+
+This phase involves configuring your parachain's runtime `/runtime/src/lib.rs` to make use of async backing system.
+
+1. Establish and ensure that constants for capacity ([`UNINCLUDED_SEGMENT_CAPACITY`](https://github.com/polkadot-fellows/runtimes/blob/d49a9f33d0ea85ce51c26c84a70b61624ec06901/system-parachains/constants/src/polkadot.rs#L42-L44){target=\_blank}) and velocity ([`BLOCK_PROCESSING_VELOCITY`](https://github.com/polkadot-fellows/runtimes/blob/d49a9f33d0ea85ce51c26c84a70b61624ec06901/system-parachains/constants/src/polkadot.rs#L45-L47){target=\_blank}) are both set to `1` in the runtime.
+
+2. Establish and ensure the constant relay chain slot duration measured in milliseconds equal to `6000` in the runtime.
+
+    ```rust title="lib.rs"
```

Comment:
Have you done code snippets yet? Using the file name in the title is excellent but, these code examples should be in snippet(s) rather than in the markdown. If you need help with creating or using snippets, please let me or Nico know. He's done a ton!